src/software/world/world.cpp

```diff
@@ -159,7 +158,7 @@ Timestamp World::getMostRecentTimestampFromMembers()
     // Add all member timestamps to a list
     std::initializer_list<Timestamp> member_timestamps = {
         friendly_team_.getMostRecentTimestamp(), enemy_team_.getMostRecentTimestamp(),
-        ball_.getPreviousStates().front().timestamp(), field_.getMostRecentTimestamp()};
+        ball_.getPreviousStates().front().timestamp()};
```

Comment:
Would be good to update the comment for `getMostRecentTimestampFromMembers` in world.h to something like

`     * Searches all timestamped member objects of world for the most recent Timestamp value`

Reply:
good idea. updated comment as suggested